authorization/invitation/service/invitation_service.go

```diff
@@ -421,16 +440,45 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 
 			err = s.Repositories().IdentityRoleRepository().Create(ctx, ir)
 			if err != nil {
-				return resourceID, err
+				return resourceID, redirectPath, err
 			}
 		}
 
+		redirectPath, e := s.getRelativeSpacePath(ctx, inviteToResource.ResourceID)
+		if e != nil {
+			return resourceID, redirectPath, err
+		}
+
 		// Delete the invitation
 		s.Repositories().InvitationRepository().Delete(ctx, inv.InvitationID)
 
-		// Return the resource ID
-		return inviteToResource.ResourceID, nil
+		// Return the resource ID and redirect path
+		return inviteToResource.ResourceID, redirectPath, nil
+	}
+
+	return resourceID, redirectPath, nil
+}
+
+func (s *invitationServiceImpl) getRelativeSpacePath(ctx context.Context, spaceID string) (string, error) {
+	var redirectPath string
+	space, err := s.Services().WITService().GetSpace(ctx, spaceID)
+	if err != nil {
+		return redirectPath, err
+	}
+
+	if space == nil {
```

Comment:
just to confirm: here the space does not exist, but you return an empty redirect path and no error ?